lib/i18n/backend/cache.rb

```diff
@@ -21,7 +21,7 @@
 # configure your own digest method via which responds to #hexdigest (see
 # http://ruby-doc.org/stdlib/libdoc/digest/rdoc/index.html):
```

Comment:
Is this still the link that should be here?

Reply:
@radar Ah, that's a good one. Let me fix that up with https://www.rubydoc.info/stdlib/openssl/OpenSSL/Digest instead.

Reply:
@radar Pushed up that fix and also pointing to the `https` version of the site instead of `http` which seemed like a good thing to fix as well :smile:.